build_tools/travis/install.sh

```diff
@@ -38,13 +38,20 @@ if [[ "$DISTRIB" == "conda" ]]; then
     # provided versions
     conda create -n testenv --yes python=$PYTHON_VERSION pip
     source activate testenv
-    conda install --yes nose pytest pytest-cov
+    conda install --yes nose pytest pytest-cov python-dateutil
     if [[ "$NUMPY" == "true" ]]; then
         conda install --yes numpy
     fi
-    if [[ "$DEPS" == "pillow" ]]; then
+    if [[ "$JPEG2000" == "true" ]]; then
+        echo cant apt-get install libopenjp2-7 libopenjp2-7-dev
```

Comment:
if you need apt-get, you need to move it in .travis.yml:

https://github.com/pydicom/pydicom/blob/master/.travis.yml#L39